Reaction, Session finding, Voice channel, Max users checkpoint

eskalVAR opened the pull request.

Commit:
Merge branch 'master' into Dev

Commit:
Added !lfgadd which adds a game and game search verification

NorbiPeti committed:
Added checks for undefined

NorbiPeti committed:
Added joining by reaction (#3)

Commit:
Added lobby searching and patched game addition

Commit:
Merge branch 'Reactions' into Dev

Commit:
Added role assignment by reaction

Commit:
Fixed typo in error message

Commit:
Removed token once again + Fixed syntax error

Commit:
Fixed error when adding game

NorbiPeti commented on src/bot.js:
```diff
@@ -137,4 +140,5 @@ process.on('unhandledRejection', err => {
-bot.login(process.env.TOKEN);
\ No newline at end of file
+//bot.login(process.env.TOKEN);
+bot.login("MzA0NTkyMDExNDkzNjM4MTQ1.C9o5ng.ZIPBx3ZFSgIYzD_zLuOoAMuuzf8");
```
Remove token 😛

Commit:
Stopped whole objects being parsed and reaction fix

NorbiPeti commented on src/bot.js:
```diff
@@ -2,110 +2,107 @@ const config = require('./config.js');
-    })
-
-}
+        MESSAGE.reply(`Error.\n ${GAME} could not be removed.`);
```
Added, not removed. 😛

eskalVAR replied:
My bad.

NorbiPeti commented on src/bot.js:
```diff
@@ -115,30 +112,27 @@ bot.on('ready', () => {
-//bot.login(process.env.TOKEN);
-bot.login("MzA0NTkyMDExNDkzNjM4MTQ1.C9o5ng.ZIPBx3ZFSgIYzD_zLuOoAMuuzf8");
\ No newline at end of file
+bot.login(process.env.TOKEN);
```
Thank you.

Commit:
Fixed typo

NorbiPeti commented on src/bot.js:
```diff
@@ -2,76 +2,107 @@ const config = require('./config.js');
+            //Adds user to existing session
+            else { 
+                config.addUser(GUILD_ID, FOUND, AUTHOR.id)
+                msg.member.addRole(FOUND)
```
msg is not defined

Commit:
Fixed msg is undefined

Commit:
Fixed getByChannelId guild not defined

LiTO773 committed:
Admins can now define the max players number per game
!lfgremove works
Role and Group name are now related to the game.

Commit:
Fixed typo that stopped channel tagging

Commit:
Fixed that stopped channel tagging

Commit:
Merge remote-tracking branch 'LiTO773/Dev' into Dev

NorbiPeti committed:
Merge branch 'Dev' of https://github.com/starsky135/LookingForGroup into Dev

Commit:
Fixed extra reaction upon creating lobby

Commit:
Merge branch 'Ruben' into Dev

Commit:
Commented out debugging console.log and added .removeRole

LiTO773 committed:
!lfg games lists all the games available
Added a few comments
Created function config.getGames for !lfg games

NorbiPeti committed:
Merge branch 'Dev' of https://github.com/starsky135/LookingForGroup into Dev

NorbiPeti committed:
Merge pull request #15 from LiTO773/Dev

!lfg games lists all the games available

NorbiPeti committed:
Merge branch 'Dev' of https://github.com/starsky135/LookingForGroup into Dev

NorbiPeti committed:
Fixed the guild data being undefined in some cases

CCninja86 committed:
Merge branch 'Dev' of https://github.com/starsky135/LookingForGroup

dowzhong committed:
Added group full detection.

We still need to make the bot proactively send a message when a group becomes full.

NorbiPeti committed:
Merge branch 'Dev' into Dev

LiTO773 committed:
Voice Channel is now also created for the session
Voice & Text channel and role are now all lowercase

LiTO773 committed:
Removed the token

CCninja86 committed:
- Experiment with max players in voice channel

CCninja86 committed:
- Create voice channel
- Set user limit from maximum players when creating voice channel for a game

CCninja86 committed:
Update bot.js

NorbiPeti committed:
Merge pull request #18 from LiTO773/Dev

Voice Channel is now also created for the session

NorbiPeti committed:
Merge branch 'Dev' into max_players

NorbiPeti committed:
Merge pull request #19 from starsky135/max_players

Max players

NorbiPeti committed:
Added check for max users and for empty config

dowzhong committed:
Fixed up detecting when a group becomes full.

If a group becomes full, it is returned in the addUser promise as 'full'. you can modify the message it sends in bot.js

dowzhong committed:
Ye

ye

NorbiPeti committed:
Merge #20 and delete test.js

dowzhong committed:
Merge branch 'master' into Dev

dowzhong committed:
refixed/added - i dont even know anymore im useless past 10pm - group full detection

:<

dowzhong committed:
Lots of shit broke yo...addUser promise now returns object follow this this struct: [gameName, maxMembers, full: boolean]

dowzhong committed:
Fixed changes requested from pull request.

NorbiPeti committed:
Fixed conflicts (hopefully) #22

NorbiPeti committed:
Applying same file versions (#22)

NorbiPeti committed:
Merge pull request #22 from cthuluhoop123/Dev

Promise passes back as 'full' if last member joins a group (once again. dont know what happened to previous fix. it was gone)

NorbiPeti committed:
Added check if data folder exists
